docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md

```diff
@@ -90,3 +90,43 @@ SilverStripe\MimeValidator\MimeUploadValidator:
       - 'text/plain'
       - 'text/calendar'
 ```
+
+## Adding new image types {#add-image-format}
+
+Silverstripe CMS support JPEG, GIF, PNG and WebP images format out of the box. Silverstripe CMS can be configured to support other less common images format (e.g.: AVIF). For this to work, your version of PHP and of the [`intervention/image` library](https://intervention.io/) must support these alternative image format.
+
+For example, this snippet can be added to the configuration of older Silverstripe CMS projects to allow them to work with WebP images. 
+
+```yml
```

Comment:
Initially, I thought of providing an example of how to add AVIF, but that's so new that only people with a specially complied version of PHP8.1 can use it. I thought WebP would be a better example even if it's only going to be relevant for people using Silverstripe CMS 4.10 or earlier.

Reply:
Fair. Even docs about AVIF would be stable eventually, and probably wouldn't get updated to say "older projects" like this does.